extension/templates/devTools/ui/loading.html

```diff
@@ -0,0 +1,8 @@
+
+<div style="margin:20px;" class="panel panel-warning">
```

Comment:
maybe not now, but we should get some of these margins and other common one offs in predefined classes @nerdpruitt ping